resources/definitions/mbot3d_grid2plus.def.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "MBot3D Grid 2+",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Magicfirm",
+        "manufacturer": "Magicfirm",
+        "file_formats": "application/x3g",
+        "machine_x3g_variant": "r1",        
+        "has_machine_quality": true,
```

Comment:
It seems that the Grid 2+ (single or dual) doesn't support its default material, PLA. As a result there are no quality levels for the user to choose from and the material selection appears orange. The reason for this is because the printer definition says that this printer has machine-specific qualities, but there are no quality profiles for this printer. If you leave out this line, the default quality profiles become available. So either remove this line (like for the Grid IV) or add some quality profiles specialised for this printer.
```suggestion
```

Reply:
Removed the has_machine_quality line.